src/opt/test/KnapsackTest.java

```diff
@@ -39,13 +45,14 @@
     private static final int NUM_ITEMS = 40;
     /** The number of copies each */
     private static final int COPIES_EACH = 4;
+    /** The maximum value for a single element */
+    private static final double MAX_VALUE = 50;

```

Comment:
In this file, I made the terms "weight" and "value" consistent. Before these changes it was using "weights" and "volumes" in a confusing way.
Specifically it used the value "volume" to indicate "value".